src/projects/EnsureThat/ExceptionMessages.cs

```diff
@@ -28,6 +28,8 @@ public static class ExceptionMessages
         public static string Strings_SizeIs_Failed { get; } = "Expected length '{0}' but found '{1}'.";
         public static string Strings_IsNotNullOrWhiteSpace_Failed { get; } = "The string can't be left empty, null or consist of only whitespaces.";
         public static string Strings_IsNotNullOrEmpty_Failed { get; } = "The string can't be null or empty.";
+
```

Comment:
I believe this extra newline diverges from the surrounding style

Reply:
Fixed.